templates/CRM/Note/Form/Note.tpl

```diff
@@ -7,8 +7,41 @@
  | and copyright information, see https://civicrm.org/licensing       |
  +--------------------------------------------------------------------+
 *}
+{* View action *}
+{if ($action eq 4)}
+  <div class="crm-block crm-content-block crm-note-view-block">
+    <table class="crm-info-panel">
+      <tr><td class="label">{ts}Subject{/ts}</td><td>{$note.subject}</td></tr>
+      <tr><td class="label">{ts}Date:{/ts}</td><td>{$note.note_date|crmDate}</td></tr>
+      <tr><td class="label">{ts}Modified Date:{/ts}</td><td>{$note.modified_date|crmDate}</td></tr>
+      <tr><td class="label">{ts}Privacy:{/ts}</td><td>{$note.privacy}</td></tr>
+      <tr><td class="label">{ts}Note:{/ts}</td><td>{$note.note|nl2br}</td></tr>
+
+        {if $currentAttachmentInfo}
+            {include file="CRM/Form/attachment.tpl"}
+        {/if}
+    </table>
+    <div class="crm-submit-buttons">
+        {crmButton class="cancel" icon="times" p='civicrm/contact/view' q="selectedChild=note&reset=1&cid=`$note.entity_id`"}{ts}Done{/ts}{/crmButton}
```

Comment:
Thanks @colemanw. I think this PR is mergeable to fix the immediate problem but noting that `cid` being entity_id means that clicking it for note comments take you to a random contact since the entity id for comments is an id in the note table.

Reply:
Yea this was a minimal-ish fix copying back deleted code but in reality that link should also depend on the entity_table & construct a link appropriately if the note is e.g. for a contribution.
OTOH notes are usually viewed in popup mode where the link does nothing at all.
@civicrm-builder Retest this please

Reply:
For contributions I couldn't see a way to view them on this form - the only way to see them is to edit the contribution.

But ok let's merge and deal with after.